js/activity.js

```diff
@@ -2571,8 +2571,11 @@ class Activity {
         };
 
         const that = this;
+        const screenWidth = window.innerWidth
         window.onresize = () => {
-            that._onResize(false);
+            if(screenWidth !== window.innerWidth ){
```

Comment:
Could you please clean this up?
if (screenWidth !== window.innerWidth) {

Reply:
Looks good. I don't think it is necessary to check for Chrome.

Reply:
> Could you please clean this up? if (screenWidth !== window.innerWidth) {

oh , but  that's the line that check if it was the width or height that was adjusted.
the code won't work as we want , if we remove the line

![resize](https://user-images.githubusercontent.com/100171190/227301905-215053b0-8cad-4aca-8583-1fc8fdf35814.PNG)